azure/batch/scripts/predict_on_catalog.py

```diff
@@ -7,6 +7,7 @@
 
 import numpy as np
 import pandas as pd
+from scipy.stats import beta  # possible new dependency, Cam
```

Comment:
This code relies on zoobot so it should have scipy https://github.com/mwalmsley/zoobot/blob/875f48561251ae3ae1b2664f538e709f6961bca9/setup.py

Also we can add new deps as required via the setup.py https://github.com/zooniverse/bajor/blob/dfab1e2aabf51ea89f01f39fd88393a079d2e345/azure/batch/setup.py#L23-L25
```suggestion
from scipy.stats import beta 
```